starlite_jwt/jwt_auth.py

```diff
@@ -173,7 +173,7 @@ def create_token(
         """
         token = Token(
             sub=identifier,
-            exp=(datetime.utcnow() + (token_expiration or self.default_token_expiration)),
+            exp=(datetime.now(timezone.utc) + (token_expiration or self.default_token_expiration)),
```

Comment:
you must ensure that timezone is backwards compatible with py3.8 and py3.7. Im not sure it is.